frontend/src/metabase/querying/filters/components/RelativeDateShortcutPicker/utils.ts

```diff
@@ -30,15 +30,15 @@ export function getShortcutGroups(): ShortcutGroup[] {
       columns: 3,
       shortcuts: [
         {
-          label: t`Week`,
+          label: ngettext(msgid`Week`, `Weeks`, 1),
```

Comment:
Maybe write a doc string explaining this unusual code?

Reply:
i think it would be spammy. I updated the PR description to explain it a little more, so people can get there via blame if they need